tfhe-zk-pok/src/proofs/rlwe.rs

```diff
@@ -763,7 +829,11 @@ pub fn verify<G: Curve>(
         if e(pi, G::G2::GENERATOR)
             != e(c_y.mul_scalar(delta_y) + c_h, c_hat)
                 - e(c_y.mul_scalar(delta_eq), c_hat_t)
-                - e(g_list[1], g_hat_list[n]).mul_scalar(t_theta * delta_theta)
+                - e(
+                    G::G1::projective(g_list[1]),
```

Comment:
some comment on the use of projective could be useful :)

Reply:
this is because the lists are stored in affine representation, since that's the faster one for multiexponentiation, but the operations require the projective representation. conversion from affine to projective is effectively a no-op

Reply:
ok no op is fine by me !